src/Umbraco.Web/Models/ContentEditing/ContentItemBasic.cs

```diff
@@ -34,6 +34,9 @@ public class ContentItemBasic : EntityBasic
         [DataMember(Name = "updater")]
         public UserBasic Updater { get; set; }
 
+        [DataMember(Name = "contentTypeId")]

```

Comment:
Thinking here was to add the Id to the response here was to avoid an additional look-up.  We do need the Id in order to construct the URL to the page for editing the related document or media type.  Without it we'd need an additional look-up with:

```
	contentResource.getById($scope.currentNode.id).then(function(data) {
		$scope.contentTypeId = data.contentTypeId;
	});
```

Becoming something like:

```
	contentResource.getById($scope.currentNode.id).then(function(contentData) {
		contentTypeResource.getByAlias(contentData.contentTypeAlias).then(function(contentTypeData) {
			$scope.contentTypeId = contentTypeData.id;
		});		
	});
```

I'm perhaps missing it elsewhere but can't see a look-up by alias in contentType.resource.js actually either, so that `getByAlias()` method would need adding.